### DISCUSSION TITLE
No "Update payment method." under "Payment information" (organization)

### DISCUSSION THREAD

**Discussion by thanasakpolrak** [community/community#56938] [New to GitHub]
Hi All,
I want to update my organization payment method.
From this help https://docs.github.com/en/billing/managing-your-github-billing-settings/adding-or-editing-a-payment-method
It suggest to click link "Update payment method." under "Payment information" 
![image](https://github.com/community/community/assets/6374096/bba51e74-2490-45c6-b4d6-b3e477ead72a)

but I cannot find the  "Update payment method." link
![image](https://github.com/community/community/assets/6374096/600eef68-42bc-4033-965e-c73122880c5f)

Any suggestion?

Regards,

**Comment by queenofcorgis** [comment]
Hi there - are you the admin for the organization?

**Comment by ytaka95** [comment]
I am in the same situation. I am the owner of my organization. On the Billing & Plans / Payment Information page, it is written as follows, but I do not want to link personal payment methods to the organization. That is very risky.

<img width="951" alt="Screenshot" src="https://github.com/community/community/assets/31610998/aa28e44d-c616-4119-887e-610d4447cc86">

**Reply by queenofcorgis** [reply]
@ytaka95 this is required because your organization account is on our Standard Terms of Service, which means that only the members of the organization owners team (such as yourself) are considered organization owners, and as such one of you will be asked to share your own billing details. If this is intended as a company account, and you have separate billing details for your company, then you'll need to move the account to our Corporate Terms of Service. This will allow you to list the company itself as an owner of the account and share company-specific billing details.

If you would like to make this change, please follow the steps outlined in the article below to do so:

[Upgrading to the Corporate Terms of Service](https://docs.github.com/organizations/managing-organization-settings/upgrading-to-the-corporate-terms-of-service)

There is no cost to this upgrade, as it is only a clarification of you and your companies relationship with us. Once you've made that change you'll have the option to enter your organization specific billing details, instead of using your personal details.
